low-level/files/opt/seagate/sspl/setup/manifest_support_bundle

```diff
@@ -255,9 +344,32 @@ class ManifestSupportBundle(object):
                 encl_dict.update({key:value})
         return encl_dict
 
+    def manifest_bundle_cleanup(self):
+        """Cleanup activity for Manifest Support Bundle"""
+        try:
+            remove_files = [self.encl_manifest, self.node_manifest, self.manifest_summary]
+            for rm_file in remove_files:
+                if os.path.exists(rm_file):
+                    os.remove(rm_file)
+        except Exception as ex:
+            print(f"Failed in manifest bundle cleanup. Error:{str(ex)}")
+
 if __name__ == "__main__":
     if not os.path.isfile(onGoingRequestPath):
         if len(sys.argv) == 3:
+            try:
+                sspl_state=open(sspl_state_file).readline().rstrip().split("=")
+                sspl_state = sspl_state[1] if len(sspl_state)==2 else ''
+                if sspl_state == 'active':
+                    from framework.platforms.realstor.realstor_enclosure import singleton_realstorencl
+                else:
+                    print("SSPL state is not active, So enclosure data will not be captured.")
+            except (FileNotFoundError, OSError) as e:
+                print(f"Failed to open the SSPL 'state.txt' file with an error '{e}',\
+                      Can't determine the SSPL state, So enclosure data also being collected.")
+                sspl_state = 'active'
```

Comment:
Is it okay to assume SSPL state as active we are not able to read SSPL state file?

Reply:
This same condition was applied in the previous code as well, if sspl_state_file is not present, it will still collect enclosure data after setting sspl_state = active manually.
@satish-seagate @thavanathan do we require any change ?

Reply:
Yes, agree with the @gargpalak048 
on line 259 we are assigning value to sspl_state. if any error occurs then we are again defining value in the exception block.